test/index.test.js

```diff
@@ -77,7 +77,15 @@ const mockVoices = [{
   LanguageName: 'Polish',
   Id: 'Maja',
   LanguageCode: 'pl-PL'
-}]
+},
+{
+  Gender: 'Male',
+  Name: 'Justin',
+  LanguageName: 'GB English',
+  Id: 'Justin',
+  LanguageCode: 'en-GB'
+}
+]
```

Comment:
Should be Indian data here, not GB English. That's why your unit tests are failing.